includes/wp-admin/class-admin.php

```diff
@@ -597,4 +601,11 @@ public static function add_plugin_settings_link( $actions ) {
 
 		return $actions;
 	}
+
+	/**
+	 * Adds metabox on wp-admin/tools.php.
+	 */
+	public static function tool_box() {
+		\load_template( ACTIVITYPUB_PLUGIN_DIR . 'templates/toolbox.php' );
```

Comment:
Tools should only be available [to users with `edit_posts` cap](https://github.com/wordpress/wordpress-develop/blob/6.7.2/src/wp-admin/menu.php#L362) in the first place.